c10/util/numa.cpp

```diff
@@ -0,0 +1,137 @@
+#include "c10/util/numa.h"
+
+#if defined(__linux__) && !defined(C10_DISABLE_NUMA) && C10_MOBILE == 0
+#include <numa.h>
+#include <numaif.h>
+#include <unistd.h>
+#define C10_NUMA_ENABLED
+#endif
+
+namespace c10 {
+
+#ifdef C10_NUMA_ENABLED
+bool IsNUMAEnabled() {
+  return numa_available() >= 0;
+}
+
+void NUMABind(int numa_node_id) {
+  if (numa_node_id < 0) {
+    return;
+  }
+  if (!IsNUMAEnabled()) {
+    VLOG(1) << "NUMA is not enabled";
```

Comment:
@ilia-cher Do you know what the motivation behind the logging here? Was it just to be screamy when someone accidentally deploys caffe2 in fbcode without numa support? I can't see most people actually finding this log message being useful.

Reply:
it's a log message to inform that NUMABind requested (numa_node_id >= 0) but is not performed, that would result in errors or low perf